### ISSUE TITLE
TimeSliderChoropleth is broken

### ISSUE THREAD

**Issue by noahbishop** [python-visualization/folium#1217]
Running the [provided example](https://github.com/python-visualization/folium/blob/master/examples/TimeSliderChoropleth.ipynb) produces a map with no changing colors/opacity.

**Comment by Conengmo** [comment]
Hi @noahbishop,

Can you give a bit more information? 

- What Python version are you using? 
- What version of folium?
- Do you get any error messages or warnings in Python?
- Do you see any error messages or warnings in your browser console?

**Comment by noahbishop** [comment]
>   What Python version are you using?

Python 3.7

>    What version of folium?

0.10.0

 >   Do you get any error messages or warnings in Python?

None

   > Do you see any error messages or warnings in your browser console?

```
Uncaught TypeError: Cannot read property 'hasOwnProperty' of undefined
    at i.setStyle (leaflet.js:5)
    at i._setLayerStyle (leaflet.js:5)
    at i.<anonymous> (leaflet.js:5)
    at i.eachLayer (leaflet.js:5)
    at i.setStyle (leaflet.js:5)
    at TimeSliderChoropleth.html:120
```

The corresponding line in the html file is:

```
macro_element_e040a831450244af8b8cd9c49e72109f.setStyle(function(feature) {
                feature.properties.style;
            });
```

**Comment by Conengmo** [comment]
Thanks, that's helpful. Someone (possibly me) will have to look into this.

**Comment by noahbishop** [comment]
I've looked into it, the problem lies [here](https://github.com/python-visualization/folium/blob/8ac5eb6d35940c47ea8ae194784c2b4ed4f53489/folium/plugins/time_slider_choropleth.py#L110).

That line 

- assumes that the data argument; the geojson string contains a style property 

- is missing a `return`.
 

Adding a `return` to that line along with a style property to the geojson string fixes the issue but the style property seems to be ignored all together. 

A quick hacky fix would be

```
if (feature.properties.style !== undefined){
      return feature.properties.style
}
else{
     return "" #Anything would work
}
```

**Comment by ak4728** [comment]
I also see the exact same error.

Python 3.6.5
Folium 0.10.0

Errors are the same as noahbishop.

**Comment by Conengmo** [comment]
@noahbishop thanks for looking into this! That's a weird piece of code, it doesn't do anything as is. Do you want to open a PR with your fix?

**Comment by Conengmo** [comment]
https://github.com/python-visualization/folium/pull/1227 which fixes this issue has been merged.

**Comment by Conengmo** [comment]
@noahbishop thanks for investigating this issue. @khllkcm thanks for making the PR. This change is now available in the v0.10.1. release that was made available just now.
